src/settings_type.h

```diff
@@ -132,7 +132,8 @@ struct GUISettings {
 	bool   timetable_arrival_departure;      ///< show arrivals and departures in vehicle timetables
 	bool   right_mouse_wnd_close;            ///< close window with right click
 	bool   pause_on_newgame;                 ///< whether to start new games paused or not
-	bool   enable_signal_gui;                ///< show the signal GUI when the signal button is pressed
+	uint8  signal_gui_mode;                  ///< select which signal types are shown in the signal GUI
+	uint8  cycle_signal_types;               ///< what signal types to cycle with the build signal tool.
```

Comment:
Sentences start with a capital ;) And dots at end of line :)

Reply:
Ah, but the rest of them don't, and changing all of them is a bit of a bigger job :p Consistency first?

Reply:
Lies: `Allow quick access to 'goto button' in vehicle orders window` <- start with capital :P

In general we said to introduce new ones correctly, and leave old ones if it is too much effort, not? Other recent examples:

`Game speed to use when fast-forward is enabled.`

etc :)

Reply:
ok, with the exception of the ~5 already inconsistent settings. Still should be fixed properly in a separate PR, IMO

Reply:
I think these two could be their enum type instead of `uint8`. Some variables do it like that, but I also see others that are `uint8` (despite being an enum), so I am not 100% sure.